### DISCUSSION TITLE
ADD - Show JPG files in Output Browser

### DISCUSSION THREAD

**Discussion by StabilityBro** [LykosAI/StabilityMatrix#479] [Ideas]
👉 This proposal suggests integrating the functionality to display JPG files directly in the Output Browser. This enhancement aims not only to facilitate result review but also to streamline the analysis and evaluation process. Including this feature would provide users with a more comprehensive and efficient experience, ultimately enhancing the overall usability of StabilityMatrix.

![Output Browser - Stability Matrix - Stable Diffusion](https://github.com/LykosAI/StabilityMatrix/assets/160883456/7a2238a0-e972-4c89-a4d7-63c7b3f66ae2)

Thank you to the entire team for your continuous efforts and dedication. Your work is instrumental in advancing the capabilities of StabilityMatrix, and your commitment to excellence is truly appreciated. Looking forward to the continued success of this remarkable project. 🙏

**Comment by lirexxx** [comment]
+1 !

I was about to open the same request but decided to check beforehand. Knowing that popular packages like Fooocus are utilizing the jpg extension should also be considered. Please add jpg support and thanks for the wonderful job you've done so far!
